app/src/docs/v1.api-spec.yaml

```diff
@@ -0,0 +1,2734 @@
+              msgId:
+                type: string
+                description: UUID for email message
+                format: uuid
+                example: b76469a3-896c-4435-aa83-b8b87db6f701
+              to:
+                type: array
+                items:
+                  type: string
+                  description: The To email addresses
+        txId:
+          type: string
+          format: uuid
+          example: b76469a3-896c-4435-aa83-b8b87db6f701
+    Response-Error:
+      title: Response Error
+      type: object
+      allOf:
+        - $ref: '#/components/schemas/Response-Problem'
+        - type: object
+          properties:
+            status:
+              example: 500
+            title:
+              example: Internal Server Error
+            type:
+              example: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/500
+    Response-Forbidden:
+      title: Response Forbidden
+      type: object
+      allOf:
+        - $ref: '#/components/schemas/Response-Problem'
+        - type: object
+          properties:
+            type:
+              example: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/403
+            title:
+              example: Forbidden
+            status:
+              example: 403
+            detail:
+              example: User lacks permission to complete this action
+    Response-NotFound:
+      title: Response Not Found
+      type: object
+      allOf:
+        - $ref: '#/components/schemas/Response-Problem'
+        - type: object
+          properties:
+            status:
+              example: 404
+            title:
+              example: Not Found
+            type:
+              example: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404
+    Response-Problem:
+      title: Response Problem
+      type: object
+      required:
+        - type
+        - title
+        - status
+      properties:
+        type:
+          type: string
+          description: What type of problem, link to explanation of problem
+        title:
+          type: string
+          description: Title of problem, generally the HTTP Status Code description
+        status:
+          type: string
+          description: The HTTP Status code
+        detail:
+          type: string
+          description: >-
+            A short, human-readable explanation specific to this occurrence of
+            the problem
+        instance:
+          type: string
+          description: >-
+            A URI reference that identifies the specific occurrence of the
+            problem
+    Response-SearchUsers:
+      title: Response for searching users.
+      type: array
+      items:
+        allOf:
+          - $ref: '#/components/schemas/DB-User'
+          - type: object
+            properties:
+              groups:
+                type: array
+                description: List of groups (associated roles) for a user
+                example: ['NAVIGATOR']
```

Comment:
Style: When specifying arrays in YAML, it is generally better to represent array contents as separated bullet points instead of the more JS style array:
```yaml
example:
  - NAVIGATOR
```